### ISSUE TITLE
[BUG] Unable to upload found photo

### ISSUE THREAD

**Issue by powellzer** [KenEucker/biketag-vue#159]
**Describe the bug**
Unable to upload found photo

**To Reproduce**
1. Go to https://philadelphia.biketag.io/#/play
2. Select Photo, enter location, enter name, Click "I Found It" Button
3. See error "ERROR: You are not the owner of album 'eNJ08S7', which means you can't add images to it. For anonymous albums, use the album deletehash."

**Expected behavior**
Would love to play the game and have it upload. I also tried creating an account with my google account and it still won't let me upload.

**Screenshots**
![IMG-1323](https://user-images.githubusercontent.com/1462955/189725710-cf84fbbd-1dc6-42ee-bffe-1c81d0fb5b09.PNG)

**Environment (please complete the following information):**
Tried on iPhone 13 and Macbook pro.

**Additional context**
This is my first time playing. Looks like fun!

**Comment by KenEucker** [comment]
@powellzer I apologize for not getting back to you sooner but I haven't been able to do any software work for a couple of months. I am told by other players that this might be resolved now. Try it again and if it is not, please let me know.

**Comment by powellzer** [comment]
Hey @KenEucker, no problem at all. I just tried to submit a biketag and I'm seeing the same error. I think this problem is specific to the https://philadelphia.biketag.io/ subdomain as I tried a test post on a different biketag city and it allowed the post to go through.
I should mention, it's possible this isn't an issue with the code itself, rather an issue with how the philadelphia instance is configured (or perhaps an issue with how the Imgur album that the philadelphia instance uses is configured).

I guess it's also possible that this was an issue with the codebase and was fixed, but maybe the philadelphia instance never received the fix? 

Either way not a huge issue, I'm just unable to play on the philly bike tag instance. I'm not totally sure how popular it is here in philly, but I recently saw a flyer for biketag at a local bike shop and decided to check it out.

Here's a stackoverflow answer that might be related to the issue: https://stackoverflow.com/a/70565591

If you know who runs the philadelphia instance I'd be happy to reach out to them to report the issue. I didn't see any contact info on the site.
